src/app/(application)/calendar/components/EventCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { IconDots } from "@tabler/icons-react";
+import React from "react";
+
+interface Props {
+  variant: "green" | "red" | "gray" | "orange" | "yellow" | "blue";
+}
+
+function EventCard({ variant }: Props) {
```

Comment:
As this is a event card component , can you pass all the stuff like title , time etc as  props. This is not necessary if you want you can do it , or else I'll merge this PR

Reply:
I can do that. I thought those things would be done later.